docs/src/pages/guides/csp.md

```diff
@@ -0,0 +1,25 @@
+# Content Security Policy
+
+Starting in JSS version 9.6.0, Material-UI supports Content Security Policy headers.
+
+## What is CSP and why is it useful?
+
+Basically, CSP mitigates cross-site scripting (XSS) attacks by requiring developers to whitelist the sources their assets are retrieved from. This list is returned as a header from the server. For instance, say you have a site hosted at `https://website.example` the CSP header `default-src: 'self';` will allow all assets that are located at `https://website.example/*` and deny all others. If there is a section of your website that is vulnerable to XSS where unescaped user input is displayed, an attacker could input:
+
+```
+<script src="https://hostile.example/bad.js"></script>
+```
+
+Although, with a secure CSP header, the browser will not load this script.
+
+You can read more about CSP [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP).
+
+## How does one implement CSP?
+
+In order to use CSP with Material-UI (and JSS), you need to set a `style-src` nonce in the meta tag like so:
+
+```
+<meta property="csp-nonce" content="longAndRandomStringGeneratedByServerUponEachRequest">
```

Comment:
This seems to imply that you just add a meta-tag to your index.html and you're good, whereas the JSS tutorial is much more involved. 

Also it's not clear how this relates to the previous example. There is some assumed knowledge here.

Reply:
There is assumed knowledge. The technicals of CSP, I’d argue, is outside the scope of this package. If you are familiar with CSP and how it works, this guide is all you need to achieve it. If you’re unfamiliar, the resources are linked.

Basically all you need it that meta tag with a random string that’s generated on each request. The tutorial on JSS uses express and react helmet (not all users of Material-UI are going to use express or helmet). My current implementation of this in my current project is actually very different than the JSS tutorial provides and all I needed to know to get it working was the information of the header.

But I’ll provide a little bit more detail on SSR and generating the nonce.